feature1/one-more-file.txt

```diff
@@ -0,0 +1,2 @@
+This file was also added in the feature1 branch
```

Comment:
```suggestion
This file was also added in the feature1 branch
```
Nice job! It is great that you decided you add this line in this file.

Reply:
Thank you for the comment.